clients/secret/input.go

```diff
@@ -0,0 +1,43 @@
+package secret
```

Comment:
(I think) you shouldn't need to add this file. Instead, you can use the `GetPassword` method on `StdIO` that's already in the repo. If your client embeds a `clients.CLI` then you should be able to do (roughly) `c.StdIO.GetPassword`.

We might want to rename the method to `GetSecret` to show it's more generic...